transformer_lens/components.py

```diff
@@ -859,14 +859,15 @@ def __init__(self, cfg: Union[Dict, HookedTransformerConfig], block_index):
             else:
                 self.mlp = MLP(cfg)
 
-        self.hook_q_input = HookPoint()  # [batch, pos, d_model]
-        self.hook_k_input = HookPoint()  # [batch, pos, d_model]
-        self.hook_v_input = HookPoint()  # [batch, pos, d_model]
+        self.hook_attn_in = HookPoint()  # [batch, pos, n_heads, d_model]
```

Comment:
Is hook_q_input through hook_v_input always [batch, pos, n_heads, d_model]? My understanding is that it's [batch, pos, n_heads, d_model] if split_qkv or attn_in, else [batch, pos, d_model]. If it might sometimes be [batch, pos, d_model] having [batch, pos, n_heads, d_model] as the comment is a bit misleading.

Reply:
Isn't this false? As far as I can see `hook_q_input` just isn't used at all is `split_qkv` is False?

To ensure we are not talking through each other, I am talking about[ the implementation on main here](https://github.com/neelnanda-io/TransformerLens/blob/feefcf1/transformer_lens/components.py#L895#L910) and I don't think I'm changing this implementation?